src/game/game_main.cpp

```diff
@@ -5,34 +5,71 @@
 
 namespace Game {
 
+const bool debug = true;
+
 static Renderer::ParticleSystem system;
+Physics::ShapeID paddle_shape;
+Physics::ShapeID ball_shape;
 
-Physics::ShapeID square;
+struct Paddle : public Logic::Entity {
+    Player player = Player::ANY;
```

Comment:
I would group these with "REGISTER_FIELDS" so you don't have to scroll to access it.

Reply:
What happens when I register fields? Documentation only specifies that the fields are "exported" (https://fog.sörnäs.se/#entitysystemaddinganewentitytype).